packages/twenty-server/src/engine/core-modules/file/file-upload/services/file-upload.service.ts

```diff
@@ -32,6 +34,31 @@ export class FileUploadService {
     });
   }
 
+  private _sanitizeFile({
+    file,
+    ext,
+    mimeType,
+  }: {
+    file: Buffer | Uint8Array | string;
+    ext: string;
+    mimeType: string | undefined;
+  }): Buffer | Uint8Array | string {
+    const fileString = file.toString();
```

Comment:
As I understand it, for every upload (even a 30MB video file upload), we would convert the buffer to a string? is that correct?

Reply:
yes, you are right, I will convert only after checking the extention and mimeType